examples/graphs/proteins_pytorch_geometric/distributed.yaml

```diff
@@ -1,22 +1,24 @@
 name: proteins_pytorch_geometric_distributed
 hyperparameters:
-  global_batch_size: 256
+  global_batch_size: 60
   dataset: PROTEINS
-  lr: 5.e-4
-  weight_decay: 1.e-4
-  hidden_channels: 32
+  lr: 1.e-3
+  topk_pooling_ratio: 0.8
+  dropout: 0.5
   training_records: 890
 records_per_epoch: 890
+min_validation_period:
+  epochs: 1
 searcher:
   name: single
   metric: validation_loss
   max_length:
-    epochs: 300
+    epochs: 200
   smaller_is_better: true
-entrypoint: model_def:GCNTrial
-resources:
-  slots_per_trial: 4
+entrypoint: model_def:GraphConvTrial
 environment:
   image:
     cpu: determinedai/environments:py-3.7-pytorch-1.9-lightning-1.3-tf-2.4-cpu-b06dafb
```

Comment:
Is there a reason we're specifying docker images instead of using the cluster defaults?

Reply:
yes, it's mentioned in the README: https://github.com/determined-ai/determined/pull/2706/files#diff-5fe53f5cccc533a2935b718527bc4c347172512022f9628b372a30d89463ddf5R15

Reply:
Missed it, mb!